### DISCUSSION TITLE
[Feature Request] Option to rename Sessions under "Manage Session"

### DISCUSSION THREAD

**Discussion by MustachedMario** [standardnotes/forum#2798] [Suggestions]
**Have you read the [Longevity Statement](https://standardnotes.org/longevity)?**

**Is your feature request related to a problem? Please describe.**
Maybe. Using the WebApp here an there. The Active Sessions filled up with a plenty of entries, or a session duplicated because the browser is updated to a new major release. It's hard to see which session entry belongs to what.

**Describe the solution you'd like**
Allow renaming or adding a comment to an active session.

**Comment by atmoio** [comment]
On our roadmap 👍

**Comment by MustachedMario** [comment]
I thought about it again. Renaming or commenting the active sessions is a good idea, but to keep an eye of orphan or duplicated sessions, you have dive into the account menu and check "manage sessions" for changes every now and then.

**Idea**: What if there was a optional switch under the security tab, that brings a popup that informs the user that the current session is a new one, so the user can name/comment/delete it later on.

I'm not sure what causes the duplicated entries. I'm pretty sure that in my case this was major browser updates, but I'm also pretty sure, that i wasn't auto logged out during these updates. So theoretical, there should no new session exists. A popup could be a solution for this.
